src/test/make/lift-directories.mk

```diff
@@ -0,0 +1,41 @@
+.EXPORT_ALL_VARIABLES:
+
+# Run from the directory basil/src/test/*/test_case/
+
+# - means continue if it doesnt exist
+-include ./config.mk
+
+NAME=$(notdir $(shell pwd))
+GIT_ROOT?=$(realpath ../../../../)
+
+GCC ?= aarch64-linux-gnu-gcc
+CLANG ?= clang
+CC ?= $(GCC)
+#CFLAGS=-fno-pic -fno-plt
+TARGET=aarch64-linux-gnu
+
+BAP?=bap
+READELF ?= aarch64-linux-gnu-readelf
+BASIL=$(GIT_ROOT)/target/scala-3.3.1/wptool-boogie-assembly-0.0.1.jar
+
+C_SOURCE ?=$(realpath $(wildcard *.c))
+SPEC ?=$(realpath $(wildcard *.spec))
+EXTRA_SPEC ?=$(realpath $(wildcard *.bpl))
+BASIL_FLAGS ?= --boogie-use-lambda-stores
```

Comment:
This shouldn't be the default? You may want this for testing the lambda stores, but make it its own thing